portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,9 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['Avoid fruits and nuts. You are what you eat', ' Chop your own wood and it will warm you twice', 
+      ' Life is far too important a thing ever to talk seriously about', 
+      'Your future depends on your dreams. So go to sleep', 'I know that you believe you understand what you think I said, but I’m not sure you realize that what you heard is not what I meant.'];
```

Comment:
It's easier to read and review changes when putting each list item on its separate line.

Also please see https://google.github.io/styleguide/jsguide.html#formatting-column-limit.